### ISSUE TITLE
Add a required ruby version

### ISSUE THREAD

**Issue by mrleeio** [sangster/extractexifgps#1]
Hi Jon,

A small but important quality of life update to your project would be to define a required Ruby version in your gemspec file.  Using jeweller I guess you would modify your Rakefile like so:

```
Jeweler::Tasks.new do |gem|
  ...
  gem.required_ruby_version = '~> 2.5'
end
```

Cheers, nice work.

**Comment by sangster** [comment]
Ah, good point!

I always include [.ruby_version](https://github.com/sangster/extractexifgps/blob/master/.ruby_version) in my projects, for the benefit of `rbenv`, `rvm`, and friends. Typically I then `IO.read` that into the gemspec file, but I neglected to do that here. Whoops!

**Comment by sangster** [comment]
Fixed in [v1.0.1](https://github.com/sangster/extractexifgps/releases/tag/v1.0.1).
